apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -151,7 +151,7 @@ static void utc_media_MediaRecorder_getVolume_p(void)
 static void utc_media_MediaRecorder_getVolume_n(void)
 {
 	MediaRecorder mr;
-	TC_ASSERT_EQ("utc_media_MediaRecorder_getVolume", mr.getVolume(), -1);
+	TC_ASSERT_EQ("utc_media_MediaRecorder_getVolume", mr.getVolume(), 0);
```

Comment:
This value should check last set value.
So I think we'd better check when last set volume called successfully. maybe below(it set as 10)
TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_ERROR);
In this case we'd better define MAX_VOLUME_LEVEL or something here and 
TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(DEFINE_VOLUME + 1), RECORDER_ERROR);

and then put that defined value instead of 0